.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,51 +1,94 @@
+      description: What Appium server version are you running?
+      placeholder: Output of `appium --version`
+    validations:
+      required: true
+
+  - type: dropdown
     attributes:
-      label: 'Is there an existing issue for this?'
-      description: 'Please [search :mag: the issues](https://github.com/appium/appium/issues) to check if this bug has already been reported.'
+      label: Appium Host Type
+      description: |
+        How is your Appium server being hosted? For options other than "Command Line",
+        please add more details under "Further Information".
       options:
-      - label: 'I have searched the existing issues'
-        required: true
+        - Command Line
+        - Virtual Machine/Docker
+        - Cloud Provider (e.g. BrowserStack/Sauce Labs)
+        - Other
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Are you using Appium components not maintained by the Appium team?
+      description: Refer to the [Appium Ecosystem](http://appium.io/docs/en/latest/ecosystem/) page for officially-supported components.
+      placeholder: Please provide the names of any third-party components.
+
   - type: textarea
     attributes:
-      label: 'Current Behavior'
-      description: 'Describe the problem you are experiencing.  **Please do not paste your logs here.**  Screenshots are welcome.'
+      label: Current Behavior
+      description: Describe the problem you are experiencing. **Please do not paste your logs here.** Screenshots are welcome.
     validations:
       required: true
+
   - type: textarea
     attributes:
-      label: 'Expected Behavior'
-      description: 'Describe what you expect to happen instead.'
+      label: Expected Behavior
+      description: Describe what you expect to happen instead.
     validations:
       required: true
+
+  - type: input
+    attributes:
+      label: Appium Log
+      description: |
+        Please create a [Gist](https://gist.github.com) with your _full_ Appium server log and link it here.
+        Alternatively, you can directly attach your logfile in the "Further Information" field below.
+
+        :warning: _Remember to redact or remove any sensitive information!_
+      placeholder: 'https://gist.github.com/...'
+
   - type: textarea
     attributes:
-      label: 'Minimal Reproducible Example'
+      label: Environment Details
       description: |
-        Please provide a the _smallest, complete code snippet_ that Appium's maintainers can run to reproduce the issue ([read more about what this entails](https://stackoverflow.com/help/minimal-reproducible-example)).  Failing this, any sort of reproduction steps are better than nothing!
+        Please provide additional information about your environment; feel free to remove any items which are not relevant.
+
+        :warning: _Remember to redact or remove any sensitive information!_
+      value: |
+          - Debug details (`appium --show-debug-info`):
```

Comment:
👍 

This value prints environment variables. Some env could include sensitive info on that so perhaps `:warning: _Remember to redact or remove any sensitive information!_` reminder would also be helpful here too (if possible)

Reply:
It's right there 2 lines higher :)
(for the record, I don't think the command output should include env vars, though I do agree that some of them may indeed be relevant for debugging purposes)